threatx/info.json

```diff
@@ -0,0 +1,353 @@
+  "icon_large_name": "large.png",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "server_url",
+        "description": "URL of the ThreatX server to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "API Key",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "password",
+        "name": "api_key",
+        "description": "The api key for the ThreatX server to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "Customer Name",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "customer_name",
+        "description": "The customer name for the ThreatX server to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "block_ip",
+      "title": "Block IP Address",
+      "description": "Add IP Address entry to blocklist in Threat X.",
+      "category": "investigation",
+      "annotation": "block_ip",
+      "output_schema": {"Ok": ""},
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "IP Address",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "ip_address",
+          "value": "",
+          "tooltip": "IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\".",
+          "description": "Specify IP address which you want to block in Threat X. IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\"."
+        },
+        {
+          "title": "Description",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "description",
+          "value": "",
+          "tooltip": "IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\".",
+          "description": "Specify the description of IP address in the block list."
+        }
+      ]
+    },
+    {
+      "operation": "unblock_ip",
+      "title": "Unblock IP Address",
+      "description": "Remove IP Address entry from blocklist in Threat X.",
+      "category": "investigation",
+      "annotation": "unblock_ip",
+      "output_schema": {"Ok": ""},
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "IP Address",
+          "required": false,
```

Comment:
This parameter is un-required? Can you please check.
Also, check for other parameters of this operation.